### PR TITLE
Null safety migration of packages/flutter_tools/test/general.shard, part 1/2

### PR DESCRIPTION
**Pull request by liamappelbe**
Null safety migration of packages/flutter_tools/test/general.shard, part 1/2

### CODE REVIEW

**Comment by liamappelbe** [comment]
Tests are passing

File: packages/flutter_tools/test/general.shard/base/logger_test.dart [hunk 1]
```diff
@@ -936,51 +932,6 @@ void main() {
       expect(lines[0], equals('All good.'));
     });
 
-    testWithoutContext('Stdout printStatus handle null inputs on colored terminal', () async {
```
Comment (christopherfujino) [inline packages/flutter_tools/test/general.shard/base/logger_test.dart:939]: Is this no longer a legitimate test?

Comment (liamappelbe) [reply packages/flutter_tools/test/general.shard/base/logger_test.dart:939]: You can't pass null to Logger.printStatus anymore.

Comment (christopherfujino) [reply packages/flutter_tools/test/general.shard/base/logger_test.dart:939]: SGTM

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**